src/commands/Utility/minecraft.ts

```diff
@@ -71,7 +71,11 @@ const command: Command = {
 		} else if (usernameInput && usernameInput.length < 32) uuid = await getUUID(usernameInput)
 		else uuid = usernameInput ?? authorDb.uuid
 		if (!userInput && !usernameInput && !authorDb.uuid) throw "noUser"
-		if (!uuid) throw "falseUser"
+		const isValidUUID = await axios
+			.get(`https://sessionserver.mojang.com/session/minecraft/profile/${uuid}`, fetchSettings)
```

Comment:
I must say, I'm not a big fan of doing another API call. You can easily check if a string is of length 32 and is a valid hexadecimal number (and if the UUID contains dashes, before anything check that the "chunks" are of valid length).

Reply:
Yeah we’re not gonna add a new API call to validate UUIDs and add that to other instances where we use them. You can also use optional chaining on the function and default to null if nullish